README.md

```diff
@@ -72,7 +94,7 @@ Stream Registry can then be started
 make run
 ```
 
-Check that the application's Swagger API is running at http://localhost:8080/swagger
+Once Stream Registry has started, check that the application's Swagger API is running at http://localhost:8080/swagger
```

Comment:
should this be `http://localhost:8080/private/swagger` ?

Reply:
Nope

```
[stream-registry_1  | [19:37:18.353] [main] INFO   i.d.j.DropwizardResourceConfig -  The following paths were found for the configured resources:
[stream-registry_1  | 
[stream-registry_1  |      GET     /swagger (io.federecio.dropwizard.swagger.SwaggerResource)
```